### ISSUE TITLE
Failing test: TestForceSystemdEnv on Windows Docker 

### ISSUE THREAD

**Issue by sharifelgamal** [kubernetes/minikube#10625]
https://storage.googleapis.com/minikube-builds/logs/10603/fb9f08b/Docker_Windows.html#fail_TestForceSystemdEnv

**Comment by hetong07** [comment]
It also has following error info, does it mean we have to increase docker memory limit?
```
W0225 21:04:42.909421   11676 out.go:191] X Docker Desktop only has 20001MiB available, you may encounter application deployment failures.
	X Docker Desktop only has 20001MiB available, you may encounter application deployment failures.
	W0225 21:04:42.909763   11676 out.go:191] * Suggestion: 
	
	    1. Open the "Docker Desktop" menu by clicking the Docker icon in the system tray
	    2. Click "Settings"
	    3. Click "Resources"
	    4. Increase "Memory" slider bar to 2.25 GB or higher
	    5. Click "Apply & Restart"
	* Suggestion: 
	
	    1. Open the "Docker Desktop" menu by clicking the Docker icon in the system tray
	    2. Click "Settings"
	    3. Click "Resources"
	    4. Increase "Memory" slider bar to 2.25 GB or higher
	    5. Click "Apply & Restart"
	   ```

**Comment by medyagh** [comment]
we could check what is the cgroup on Windows's Docker Desktop

**Comment by spowelljr** [comment]
Fixed with #11804
